metadata.rb

```diff
@@ -11,6 +11,7 @@
 chef_version '>= 12.14.34'
 
 depends 'ark'
+depends 'build-essential'
```

Comment:
is that required? I think you had a berkshelf bug. If you don't rely on build-essential directly, don't add it.

Reply:
Berkshelf bug was fixed, now removed this dependency.